formapp/app.py

```diff
@@ -68,8 +74,9 @@
 flasklog = logging.getLogger('werkzeug')
 flasklog.setLevel(logging.ERROR)
 
-#Display IP address to console for user connection
-print("[info] " +'Running formbar server on:' + ip)
+#Display IP address to console for user connection. Updates the Time as well
+now = datetime.datetime.now()
+print("[" + now.strftime("%Y-%m-%d %H:%M:%S") + "]" + " [info] " +'Running formbar server on:' + ip)
```

Comment:
Combine these wherever you see them.
`print("[" + datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S") + "]" + " [DEBUG] " + str(message))`
Consider using a function to reduce the amount of overall code on the screen.